addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -227,6 +474,10 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
 
         // Adapt size array preview to fit ppr
         $size.find('tr td:nth-child(n + ' + parseInt(this.ppr + 1) + ')').hide();
+        if (this.rerender) {
+            this.rerender = false;
+            return this._rerenderXML();
+        }
     },
```

Comment:
You actually did it in fact, why not `this.rerender = true` where needed then? ^^

Reply:
I think I just got distracted in the middle of implementing that ^^'